### ISSUE TITLE
Build from source fails after an upgrade with error referencing libsnark::div_ceil

### ISSUE THREAD

**Issue by prettyCoders** [zcash/zcash#3624]
![image](https://user-images.githubusercontent.com/36216120/47480529-51c49600-d863-11e8-977c-d4af5ace12db.png)
cmd:
git fetch origin
git checkout v2.0.1
./zcutil/fetch-params.sh
./zcutil/build.sh

**Comment by daira** [comment]
Try `make clean` before `./zcutil/build.sh`.

We'll fix the documentation.

**Comment by prettyCoders** [comment]
OK!I`ll try it

**Comment by Eirik0** [comment]
Just to note - I was able to recreate this by doing the following:
```
git checkout v1.1.12
./zcutil/build.sh
git checkout v2.0.0
./zcutil/build.sh
```

**Comment by zhanghuahua0708** [comment]
> Try `make clean` before `./zcutil/build.sh`.
> 
> We'll fix the documentation.

can't make , there is no makefile

**Comment by mdr0id** [comment]
Fair enough. This was written for a case assuming you had an existing built
stack. I'll update this soon.

**Comment by daira** [comment]
Ah, I guess we do need to fix #3625 in order to address this, then.

**Comment by daira** [comment]
```export BUILD="$(./depends/config.guess)"

**Comment by daira** [comment]
Try this:
```
HOST="$(./depends/config.guess)" BUILD="$(./depends/config.guess)" make -C ./depends/ V=1
./autogen.sh
make clean
./zcutil/build.sh
```

**Comment by daira** [comment]
This *may* be fixed (if the error only relates to libsnark, rather than that just being the first error encountered).

**Comment by daira** [comment]
libsnark is long gone as a zcashd dependency, and the underlying problem with building from source after a version update is fixed by #5871.
